csm_web/scheduler/email/quickstart.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Is this file necessary?

Reply:
Quickstart.py is for generating the token.json from credentials.json and opens a Google authentication tab to authorize access. Technically it needs to be there to generate token.json the first time, but it could be removed for actual deployment. If we are switching token.json to an environment variable (and deleting credentials.json which is just used for authentication), we could probably delete it, but if we need to ever authenticate again, it is needed.

Reply:
The script is also copied from GmailAPI's quickstart tutorial.